static/monorail-2.0.yaml

```diff
@@ -3342,18 +3343,18 @@ paths:
         type: string
       responses:
         201:
-          description: Successfully created the template
+          description: Successfully created or updated a specific template.
           schema:
             type: object
         404:
-          description: The template with specified identifier was not found
+          description: The specified template was not found.
           schema:
             $ref: '#/definitions/Error'
         default:
           description: Unexpected error
           schema:
             $ref: '#/definitions/Error'
-      summary: Put a single template
+      summary: Put a specific template
```

Comment:
To align the description for other PUT method, suggest changing to "Create or update a specific template"